lang/c++/api/Stream.hh

```diff
@@ -22,6 +22,7 @@
 #include <cstdint>
 #include <cstring>
 #include <memory>
+#include <vector>
```

Comment:
Sorry I currently have a limited access to my laptop (until Friday). I'll double check this by next week. 
IIRC I got compile error without it because I removed include of boost/format from one of the headers. But I'll definitely double check

Reply:
@martin-g so yeah this include is needed, because there's usage of `std::vector` in the signature of `snapshot()` on line 177. Without it, compilation failed for me.